components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py

```diff
@@ -112,3 +113,30 @@ def test_images_from_rois(self):
         # check the result
         assert img_from_rois is not None
         assert img_from_rois.getValue().getId().getValue() > 0
+
+    def test_dataset_to_plate(self):
+        script_id = super(TestUtilScripts, self).get_script(dataset_to_plate)
+        assert script_id > 0
+        # root session is root.sf
+        session = self.root.sf
+        client = self.root
+
+        # create several test images in a dataset
+        dataset = self.make_dataset("dataset_to_plate-test", client=client)
+
+        for i in range(10):
+            # x,y,z,c,t
+            image = self.create_test_image(100, 100, 1, 1, 1, session)
+            self.link(dataset, image, client=client)
+
+        # run the script twice. First with all args...
+        dataset_ids = [omero.rtypes.rlong(dataset.getId().getValue())]
+        args = {
+            "Data_Type": omero.rtypes.rstring("Dataset"),
+            "IDs": omero.rtypes.rlist(dataset_ids)
+        }
+
+        d_to_p = run_script(client, script_id, args, "New_Object")
+        # check the result
+        assert d_to_p is not None
+        assert d_to_p.getValue().getId().getValue() > 0
```

Comment:
Maybe nice to check that a Plate got created and it has some Wells & Images?
E.g. 
```
qs = client.getSession().getQueryService()
query = "select well from Well as well left outer join fetch well.plate as pt left outer join fetch well.wellSamples as ws left outer join fetch ws.image as img where well.plate.id = :oid"
params = omero.sys.ParametersI()
params.addLong('oid', omero.rtypes.rlong(plateId))
wells = qs.findAllByQuery(query, params)
assert len(wells) > 0       # or however many we're expecting
# Some check that Wells have Images, E.g.
for w in wells:
    print w.getWellSample(0).getImage().id.val
```